react/src/components/Systems/SystemSelect.tsx

```diff
@@ -0,0 +1,53 @@
+import React, { useEffect } from 'react';
+import { useSystems } from '../../hooks';
+import { System } from '../../types';
+
+interface SystemSelectProps {
+  onSystemSelect: (selectedSystem: string) => void;
+}
+
+export const SystemSelect: React.FC<SystemSelectProps> = ({
+  onSystemSelect,
+}) => {
+  const { data: systems } = useSystems();
+
+  // use dsProjects hook here
```

Comment:
👍  in angular hazmapper, the systems list was used as all projects start with "projects-".  but using the DS projects hook makes more sense 👍